plugins/station_master.py

```diff
@@ -101,17 +108,22 @@ def boot(self):
 
     def format_train(self, train, data):
         if data['canceled']:
-            txt = "est " + self.bot.text.red("annulé")
+            status_txt = "est " + self.bot.text.bold(self.bot.text.red("annulé ❌"))
         elif data['delay'] > 0:
-            txt = 'a un ' + self.bot.text.purple('retard de %s min' % data['delay'])
+            status_txt = 'a un ' + self.bot.text.red('retard de %s min' % data['delay'])
```

Comment:
Maybe use yellow here ? Green=ok; Yellow=delay; Red=cancelled would be better imho

Reply:
Ok 👍